TripHouse/index.html

```diff
@@ -189,15 +187,259 @@ <h2 class="homes-guests-text">Homes guests loves</h2>
+        <svg class="button-dest-right" width="48" height="48">
           <use href="../images/svg-sprite.svg#Button-homes"></use>
         </svg>
+      </a>
+    </section>
+    <section class="sign-up-cont">
+      <div class="close-link-cont">
+        <button class="close-link" type="submit">
+          <svg class="close" width="26" height="26">
+            <use href="../images/svg-sprite.svg#close"></use>
+          </svg>
+        </button>
+      </div>
+      <div class="sign-up-content">
+        <svg class="user-sign-up" width="86" height="86">
+          <use href="../images/svg-sprite.svg#user-sign-up"></use>
+        </svg>
+
+        <div class="sign-up-right-content">
+          <p class="sign-up-text">Sign up and get 20% off your first booking</p>
+          <button class="sign-up-btn" type="submit">Sign up</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="guests-reviews-cont">
+      <h2 class="guests-heading">Guests reviews</h2>
+      <h2 class="guests-heading-2">Reviews</h2>
+
+      <div class="reviews-content Ukraine">
+        <div class="first-col-reviews">
+          <div class="first-img-cont">
+            <div class="guests-wrapper">
+              <p class="price-hotel">From 250 USD</p>
+              <img
+                class="guests-img"
+                src="../images/bali.svg"
+                alt="Ubud Bali Resort&SPA"
+              />
+            </div>
+          </div>
+          <div class="second-col-reviews">
+            <div class="second-col-content">
+              <p class="text-review hotel">Ubud Bali Resort&SPA</p>
+              <p class="text-review mark">9,4</p>
+            </div>
+            <div class="second-col-2content">
+              <p class="text-review city">Bali, Indonesia</p>
+              <p class="text-review amount-reviews">1324 reviews</p>
+            </div>
+          </div>
+          <div class="third-col-reviews">
+            <svg class="user-logo-guests" width="54" height="54">
+              <use href="../images/svg-sprite.svg#user-logo-guests"></use>
+            </svg>
+            <div class="third-col-content">
+              <p class="text-review hotel name">Hanna Ivanova</p>
+              <div class="country-cont">
+                <svg class="country-review" width="20" height="19">
+                  <use href="../images/svg-sprite.svg#ukraine"></use>
+                </svg>
+                <p class="text-country">Ukraine</p>
+              </div>
+            </div>
+          </div>
+          <div class="review-text">
+            <p class="review">
+              Great location, really pleasant and clean rooms, but the thing
+              that makes this such a good place to stay are the staff. All of
+              the people are incredibly helpful and generous with their time and
+              advice.
+            </p>
+          </div>
+          <div class="navigation">
```

Comment:
You didn't add an arrow button for navigation on desktops
<img width="467" alt="изображение" src="https://user-images.githubusercontent.com/25488411/162590631-a8d933f8-f10d-4c82-a638-c1b36d359f80.png">


And it's better to use button instead of svg for this dots navigation

I recommend to put navigation dots in the end of section. Maybe it looks good, however in real life this example with navigation after the first slide will not work

Open devtools for Slider example to see how usually slider looks like

[Slider example](https://owlcarousel2.github.io/OwlCarousel2/demos/basic.html)